improver/cli/nowcast_accumulate.py

```diff
@@ -59,6 +60,9 @@ def process(cube: cli.inputcube,
             The input Cube to be processed.
         advection_velocity (iris.cube.CubeList):
             Advection cubes of U and V.
+            These must have the names of either:
+            precipitation_advection_x_velocity or grid_eastward_wind
+            precipitation_advection_y_velocity or grid_northward_wind
```

Comment:
@cgsandford We now have three nowcasting CLIs that take wind in different forms. The orographic enhancement CLI is taking wind speed and direction, and then calling the ResolveWindComponents plugin directly. This nowcast_accumulate CLI and the nowcast_extrapolate CLI are relying on these components having been calculated in a previous step in the suite.

Is it within your plans to make the interfaces to these CLIs more alike? i.e. use the eastward and northward wind components as input to the orographic enhancement CLI.